docs/jenkins.md

```diff
@@ -333,4 +333,4 @@ And that is the life cycle of the testing instances in https://jenkinsci.saltsta
 
 Jenkins however uses `test`, which will create, then converge, then verify, and if verify passes, the instance will be deleted, otherwise Jenkins delete it in a clean up command.
```

Comment:
We actually, in jenkins, go through each steps ourselves, we don't use `test`.

Reply:
updated. thanks